examples/chess/methods/guest/Cargo.lock

```diff
@@ -2,12 +2,198 @@
 # It is not intended for manual editing.
 version = 3
 
+[[package]]
+name = "ahash"
+version = "0.8.6"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "91429305e9f0a25f6205c5b8e0d2db09e0708a7a6df0f42212bb56c32c8ac97a"
+dependencies = [
+ "cfg-if",
+ "once_cell",
+ "version_check",
+ "zerocopy",
+]
+
 [[package]]
 name = "anyhow"
 version = "1.0.69"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "224afbd727c3d6e4b90103ece64b8d1b67fbb1973b1046c2281eed3f3803f800"
 
+[[package]]
```

Comment:
The problem is that the verifier should be included in the `default-features=false`. Since groth16 now is part of a receipt, that would not be possible to make it optional without breaking the verifier completeness

Reply:
The `Receipt` datatype is not included when `target_os = "zkvm"`

https://github.com/risc0/risc0/blob/bcfc9c6a55d00eb4696d3c9e20aff0bcff11f8ef/risc0/zkvm/src/lib.rs#L82-L90

More generally, we don't actively support directly verifying `Receipts` in the guest.